.pre-commit-config.yaml

```diff
@@ -1,5 +1,7 @@
 - repo: git://github.com/antonbabenko/pre-commit-terraform
-  rev: v1.16.0
+  rev: v1.19.0
   hooks:
     - id: terraform_fmt
     - id: terraform_docs
+    - id: terraform_validate
```

Comment:
I don't use to run `terraform validate` on every commit/rebase. Is this quick ? If not maybe this can be done only on `push` stage ?

@max-rocket-internet please advise.

Reply:
I think it's fast, it's just checking syntax

Reply:
yeah, I am using it with my projects, it would be nice to have it.

I do notice it is using `git://` though, we might want to switch to `https://`